src/Microsoft.AspNet.Authentication.Facebook/FacebookAuthenticationMiddleware.cs

```diff
@@ -37,11 +37,11 @@ public class FacebookAuthenticationMiddleware : OAuthAuthenticationMiddleware<Fa
         {
             if (string.IsNullOrWhiteSpace(Options.AppId))
             {
-                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Exception_OptionMustBeProvided, nameof(Options.AppId)));
+                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Exception_OptionMustBeProvided, nameof(Options)));
```

Comment:
Look to this line of code https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.Facebook/FacebookAuthenticationMiddleware.cs#L38